sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/SpannerReadIT.java

```diff
@@ -193,6 +184,59 @@ public void testQuery() throws Exception {
     p.run();
   }
 
+  private SpannerConfig createSpannerConfig() {
+    return SpannerConfig.create()
+        .withProjectId(project)
+        .withInstanceId(options.getInstanceId())
+        .withDatabaseId(databaseName);
+  }
+
+  @Test
+  public void testReadAllRecordsInDb() throws Exception {
+    DatabaseClient databaseClient = getDatabaseClient();
+
+    List<Mutation> mutations = new ArrayList<>();
```

Comment:
I referred also to this part too, and what I expected was more in the line of a makeTableData method like [BigtableIOTest's one](https://github.com/apache/beam/blob/50a84326581941bc1edf573a0ad2b798ecb0f6a1/sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java#L995).

Reply:
Done